i18n: add Italian translation

Misairuzame opened the pull request:
This PR adds Italian translations to Ghostty.

kristoff-it reviewed (approved):
Good Italian translation!

00-kat commented:
There are currently two Italian translations, this one and #7391; as only one can be merged, would you mind reaching a consensus on which one would be more preferable to merge?

trag1c commented:
Let's proceed with this one then!

@Misairuzame @Atomk @kristoff-it (and @domenicomastrangelo @fcurella via #7391) do any of you volunteer to be on the localization team for this locale to be pinged on string changes in the future? Ideally we'd have at least two people :)

trag1c commented:
> Some time has passed since I made this PR and I can see the base .pot has been updated and reordered, would you like me to sync my fork and recreate my translations with the new template?

Yes please.

Misairuzame committed:
Add it_IT translations

Misairuzame committed:
Change 'Split up, down, left, right' translations

Misairuzame commented:
Woah - didn't realize syncing my fork would automatically close this PR.
I have now pushed my new translations, incorporating feedback from Atomk.
I implemented one other change: I translated "Split up/down/left/right" as "Dividi in alto/in basso/a sinistra/a destra", which to me sounds better than my old translation "Dividi sopra/sotto/sinistra/destra". If you have any feedback, please let me know.
I'm on KDE and I can see they use "Dividi in verticale / in orizzontale", but that doesn't really fit here.

jcollie commented:
> I'm surprised this is still open despite other translation PRs being merged, is this supposed to make it in 1.2.0?

There's still time to make it into 1.2.0. We need to make sure that all the strings are up-to-date (there have been some last minute additions) and that we get at least a couple of fluent speakers onboard as reviewers.

Atomk commented:
I don't use Ghostty (yet, I guess) but Misairuzame did a splendid job on the translations in my opinion and it would a pity to let this PR sit here for even longer, if this unblocks the process and you're okay with me just being a mere reviewer I'll volunteer to double-check that the translations are accurate.

00-kat commented:
> if this unblocks the process and you're okay with me just being a mere reviewer I'll volunteer to double-check that the translations are accurate.

That would definitely be appreciated!

Misairuzame committed:
Merge branch 'ghostty-org:main' into main

Misairuzame committed:
Update Italian translation

Misairuzame commented:
Hi everyone,
First of all thank you @Atomk for your kind words and for volunteering to help in the future! That's awesome!

I updated my translations adding the three ones that were recently added and were missing.

I also updated a translation at line 117: "Split", which in this context I think it's used as a noun and not as a verb, in Italian should be the noun "Divisione" instead of the verb "Dividi". I looked around in other translations and I see many of them translated it as a verb.
However, this particular string is the one you see in the context menu when right-clicking a Ghostty window, and then it expands into the options to split up/down/left/right or change its title. I'd say using the noun is consistent with the other context menu options, such as "Tab" and "Window", but please correct me if I'm wrong - I'll update it ASAP.

Hopefully now this translation will make it into 1.2.0!

trag1c commented:
Hi @Atomk, could you take another look?

trag1c reviewed (approved):
Thank you!

Misairuzame commented:
@00-kat Sure, I'm still happy to contribute to translations! I'm going to take a look at the 1.3 translations in the following days.
